web/src/components/App.js

```diff
@@ -39,44 +29,22 @@ export class App extends Component {
   };
 
   render() {
-    const onClick = () => {
```

Comment:
prefer deconstructing props `const { userId, idps, authentication } = this.props`